src/popup/pages/MultisigProposalDetails.vue

```diff
@@ -52,7 +52,7 @@
                 <CopyText
                   hide-icon
                   :value="multisigTx.recipientId"
-                  :copied-text="$t('hashCopied')"
+                  :copied-text="$t('addressCopied')"
```

Comment:
I think we should not keep any common translation in the root of the translation file. I suggest to move it to `common.addressCopied`.

Reply:
I think we can create a separate task for that.